pkg/reconciler/jbsconfig/jbsconfig.go

```diff
@@ -34,11 +34,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-const TlsServiceName = v1alpha1.CacheDeploymentName + "-tls"
-const TestRegistry = "jvmbuildservice.io/test-registry"
-const ImageRepositoryFinalizer = "jvmbuildservice.io/quay-repository-finalizer"
-const DeleteImageRepositoryAnnotationName = "image.redhat.com/delete-image-repo"
-const UploadSecretName = "jvm-build-service-temp-upload-secret" //#nosec
+const (
+	TlsServiceName                      = v1alpha1.CacheDeploymentName + "-tls"
+	TestRegistry                        = "jvmbuildservice.io/test-registry"
+	JBSConfigFinalizer                  = "jvmbuildservice.io/jbsconfig-finalizer"
```

Comment:
Looks like this is left over from the previous attempt? With the ownership model you should not need the finalizer?

Reply:
I kept that bit to ensure that `handlePossibleRepositoryCleanup` would be called

Reply:
That is handled by the ImageRepositoryFinaliser below though?

Reply:
Oh, you changed the name, I guess the one below is the one that could be removed.

Reply:
Actually I just had a look at the existing ImageRepositoryFinaliser code looks like it is still all that is needed.